src/azure-cli/azure/cli/command_modules/appconfig/_params.py

```diff
@@ -79,6 +79,21 @@ def load_arguments(self, _):
         help='Name of the App Configuration store. You can configure the default name using `az configure --defaults app_configuration_store=<name>`.',
         configured_default='app_configuration_store'
     )
+
+    store_creation_replica_name_arg_type = CLIArgumentType(
+        options_list=['--replica-name'],
+        type=str,
+        help='Name of the replica of the App Configuration store.',
+        configured_default=None
+    )
+
+    replica_location_arg_type = CLIArgumentType(
+        options_list=['--replica-location'],
+        type=str,
+        help='The location of the replica of the App Configuration store.',
+        configured_default=None
+    )
+
```

Comment:
It is not necessary to specify `type=str`, since it is set to string by default
```suggestion
    store_creation_replica_name_arg_type = CLIArgumentType(
        options_list=['--replica-name'],
        help='Name of the replica of the App Configuration store.',
        configured_default=None
    )

    replica_location_arg_type = CLIArgumentType(
        options_list=['--replica-location'],
        help='The location of the replica of the App Configuration store.',
        configured_default=None
    )

```